lib/src/tabs/Tab.tsx

```diff
@@ -0,0 +1,182 @@
+  text-transform: ${(props) => props.theme.fontTextTransform};
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  flex-shrink: 0;
+  line-height: 1.75;
+  white-space: normal;
+  letter-spacing: 0.02857em;
+  border: 0;
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+  vertical-align: middle;
+  justify-content: center;
+  opacity: 0.7;
+  border-bottom-style: solid;
+  border-bottom-color: ${(props) => props.theme.dividerColor};
+  font-family: ${(props) => props.theme.fontFamily};
+  font-size: ${(props) => props.theme.fontSize};
+  font-style: ${(props) => props.theme.fontStyle};
+  font-weight: ${(props) => props.theme.fontWeight};
+  min-width: 90px;
+  max-width: 360px;
+  padding: ${(props) =>
+    ((!props.hasLabelAndIcon || (props.hasLabelAndIcon && props.iconPosition !== "top")) && "12px 16px") || "8px 16px"};
+  height: ${(props) =>
+    ((!props.hasLabelAndIcon || (props.hasLabelAndIcon && props.iconPosition !== "top")) && "48px") || "72px"};
+  min-height: ${(props) =>
+    ((!props.hasLabelAndIcon || (props.hasLabelAndIcon && props.iconPosition !== "top")) && "48px") || "72px"};
+  background-color: ${(props) => props.theme.unselectedBackgroundColor};
+  color: ${(props) => props.theme.unselectedFontColor};
+  svg {
+    color: ${(props) => props.theme.unselectedIconColor};
+  }
+
+  &:hover {
+    background-color: ${(props) => `${props.theme.hoverBackgroundColor} !important`};
+  }
+
+  &:active {
+    background-color: ${(props) => `${props.theme.pressedBackgroundColor} !important`};
+    font-weight: ${(props) => `${props.theme.pressedFontWeight} !important`};
+  }
+
+  &:focus {
+    outline: ${(props) => props.theme.focusOutline} auto 1px;
+  }
+
+  &[aria-selected="true"] {
+    background-color: ${(props) => props.theme.selectedBackgroundColor};
+    color: ${(props) => props.theme.selectedFontColor};
+    svg {
+      color: ${(props) => props.theme.selectedIconColor};
+    }
+    opacity: 1;
+  }
+
+  &[aria-disabled="true"] {
+    background-color: ${(props) => props.theme.unselectedBackgroundColor} !important;
+    cursor: not-allowed !important;
+    pointer-events: all;
+    color: ${(props) => props.theme.disabledFontColor};
+    font-style: ${(props) => props.theme.disabledFontStyle};
+    svg {
+      color: ${(props) => props.theme.disabledIconColor};
+    }
+    outline: none !important;
+    opacity: 0.5;
+  }
+`;
+
+const ParentLabelSpan = styled.div`
+  position: relative;
+`;
+
+const TabLabelContainer = styled.div<IconProps>`
+  display: flex;
+  flex-direction: ${(props) => (props.hasLabelAndIcon && props.iconPosition === "top" && "column") || "row"};
+  align-items: center;
+`;
+
+const LabelTextContainer = styled.div``;
+
+const BadgeContainer = styled.div<IconProps>`
+  position: absolute;
+  right: 0;
+  top: ${(props) => (props.hasLabelAndIcon && props.iconPosition === "top" && "0") || "5px"};
+  width: 23px;
+  height: 17px;
+`;
```

Comment:
Why these odd values `width` and `height`? Are they needed?

Reply:
Removed values. Don't know why it was decided to use these values, I agree they are odd. They are even defined in the tokens.
![image](https://user-images.githubusercontent.com/34685461/190143235-dcbaea4c-2907-44ea-b675-3ca3b1a04b3a.png)